### PR TITLE
docs(secret-manager): faq

### PR DESCRIPTION
**Pull request by nedaehm**

### CODE REVIEW

File: faq/secret-manager.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Secret Manager allows you to control access to secrets through IAM policies. When you retrieve a secret, Secret Manager decrypts it and transmits it securely via TLS to your local environment. At no point does the service write or cache secrets in plain text on permanent storage.
+
+## How does Scaleway Secret Manager Scaleway encrypt my secrets?
+
+Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
+To understand the secrets encryption process, refer to our [related documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
+
+## How is Scaleway Secret Manager billed?
+
```
Comment (bene2k1) [inline faq/secret-manager.mdx:37]:
```suggestion
```

File: faq/secret-manager.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.
+
+
```
Comment (bene2k1) [inline faq/secret-manager.mdx:41]:
```suggestion
```

**Review by bene2k1 — APPROVED** [review]

File: faq/secret-manager.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## How does Scaleway Secret Manager Scaleway encrypt my secrets?
+
+Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
+To understand the secrets encryption process, refer to our [related documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
+
+## How is Scaleway Secret Manager billed?
+
+At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.
+
+## How do I migrate my secrets to Secret Manager Scaleway?
```
Comment (RoRoJ) [inline faq/secret-manager.mdx:40]:
```suggestion
## How do I migrate my secrets to Scaleway's Secret Manager?
```

File: faq/secret-manager.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+The Secret Manager service is deployed in all Scaleway regions: PAR, WAW and AMS. Stored secrets are replicated in different zones within a region.
+
+## How does Secret Manager guarantee the security of my secrets?
+
+Secret Manager allows you to control access to secrets through IAM policies. When you retrieve a secret, Secret Manager decrypts it and transmits it securely via TLS to your local environment. At no point does the service write or cache secrets in plain text on permanent storage.
+
+## How does Scaleway Secret Manager Scaleway encrypt my secrets?
+
+Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
+To understand the secrets encryption process, refer to our [related documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
```
Comment (nedaehm) [inline faq/secret-manager.mdx:34]:
```suggestion
To understand the secrets encryption process, refer to our [related documentation](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
```

File: faq/secret-manager.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## How is Scaleway Secret Manager billed?
+
+At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.
+A secret version is billed if it is in an enable or disable state.
+If you have used a secret version for five days, you will only be billed for the five days and not for the whole month.
```
Comment (nedaehm) [inline faq/secret-manager.mdx:40]:
```suggestion
If you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out [pricing information](https://www.scaleway.com/en/pricing/) about Secret Manager.
```

File: faq/secret-manager.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+## How does Scaleway Secret Manager Scaleway encrypt my secrets?
+
+Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
+To understand the secrets encryption process, refer to our [related documentation](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
+
+## How is Scaleway Secret Manager billed?
+
+At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.
+A secret version is billed if it is in an enable or disable state.
+If you have used a secret version for five days, you will only be billed for the five days and not for the whole month.
```
Comment (nedaehm) [inline faq/secret-manager.mdx:40]:
```suggestion
If you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out more about pricing on our [dedicated page](https://www.scaleway.com/en/pricing/?tags=available).
```

File: faq/secret-manager.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+## How is Scaleway Secret Manager billed?
+
+At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.
+
+## How do I migrate my secrets to Secret Manager Scaleway?
+
+**To migrate your existing secrets to Secret Manager:**
+
+1. Identify the secrets and their associated permissions
+2. Create the API, CLI or Console secrets.
```
[the remark this answers is not in the document]
Comment (nedaehm) [reply faq/secret-manager.mdx]: I removed this part because it was not ready

File: faq/secret-manager.mdx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+## How does Scaleway Secret Manager Scaleway encrypt my secrets?
+
+Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
+To understand the secrets encryption process, refer to our [related documentation](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
+
+## How is Scaleway Secret Manager billed?
+
+At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.
+A secret version is billed if it is in an enable or disable state.
```
Comment (RoRoJ) [inline faq/secret-manager.mdx:39]:
```suggestion
A secret version is billed if it is in an enabled or disabled state.
```

File: faq/secret-manager.mdx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+If you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out more about pricing on our [dedicated page](https://www.scaleway.com/en/pricing/?tags=available).
```
Comment (RoRoJ) [inline faq/secret-manager.mdx:40]:
```suggestion
If, for example, you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out more about pricing on our [dedicated page](https://www.scaleway.com/en/pricing/?tags=available).
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**